casper/src/test/scala/coop/rchain/casper/batch1/MultiParentCasperMergeSpec.scala

```diff
@@ -59,45 +46,38 @@ class MultiParentCasperMergeSpec
     }
   }
 
-  it should "handle multi-parent blocks correctly when they operate on stdout" in effectTest {
-    def echoContract(no: Int) =
-      Rho(s"""new stdout(`rho:io:stdout`) in { stdout!("Contract $no") }""")
-    merges(echoContract(1), echoContract(2), Rho("Nil"))
-  }
-
-  it should "not conflict on registry lookups" in effectTest {
-    val uri         = "rho:id:i1kuw4znrkazgbmc4mxe7ua4s1x41zd7qd8md96edxh1n87a5seht3"
-    val toSign: Par = ETuple(Seq(GInt(0), GString("foo")))
-    val toByteArray = Serialize[Par].encode(toSign).toArray
-    val sig         = Secp256k1.sign(Blake2b256.hash(toByteArray), ConstructDeploy.defaultSec)
-    val setup =
-      Rho(s"""
-             |new rs(`rho:registry:insertSigned:secp256k1`) in {
-             |  rs!(
-             |    "${ConstructDeploy.defaultPub.bytes.toHex}".hexToBytes(),
-             |    (0, "foo"),
-             |    "${sig.toHex}".hexToBytes(),
-             |    Nil
-             |  )
-             |}
-        """.stripMargin)
-    val lookup =
-      Rho(s"""
-             |new rl(`rho:registry:lookup`) in {
-             |  rl!(`$uri`, Nil)
-             |}
-        """.stripMargin)
-
-    merges(lookup, lookup, setup)
-  }
-
-  it should "respect mergeability rules when merging blocks" ignore effectTest {
-    baseMergeabilityCases.map(_._2).parSequence
-  }
-
-  it should "respect mergeability rules when merging blocks containing events with peek" ignore effectTest {
-    peekMergeabilityCases.map(_._2).parSequence
-  }
+  // TODO enable tests below
+//  it should "handle multi-parent blocks correctly when they operate on stdout" in effectTest {
+//    def echoContract(no: Int) =
+//      Rho(s"""new stdout(`rho:io:stdout`) in { stdout!("Contract $no") }""")
+//    merges(echoContract(1), echoContract(2), Rho("Nil"))
+//  }
+//
+//  it should "not conflict on registry lookups" in effectTest {
+//    val uri         = "rho:id:i1kuw4znrkazgbmc4mxe7ua4s1x41zd7qd8md96edxh1n87a5seht3"
+//    val toSign: Par = ETuple(Seq(GInt(0), GString("foo")))
+//    val toByteArray = Serialize[Par].encode(toSign).toArray
+//    val sig         = Secp256k1.sign(Blake2b256.hash(toByteArray), ConstructDeploy.defaultSec)
+//    val setup =
+//      Rho(s"""
+//             |new rs(`rho:registry:insertSigned:secp256k1`) in {
+//             |  rs!(
+//             |    "${ConstructDeploy.defaultPub.bytes.toHex}".hexToBytes(),
+//             |    (0, "foo"),
+//             |    "${sig.toHex}".hexToBytes(),
+//             |    Nil
+//             |  )
+//             |}
+//        """.stripMargin)
+//    val lookup =
+//      Rho(s"""
+//             |new rl(`rho:registry:lookup`) in {
+//             |  rl!(`$uri`, Nil)
+//             |}
+//        """.stripMargin)
+//
+//    merges(lookup, lookup, setup)
+//  }
```

Comment:
Add reference to an issue where is tracked.

Reply:
https://github.com/rchain/rchain/issues/3277#issuecomment-798109618